spec/unit/models/token_details_spec.rb

```diff
@@ -80,6 +80,20 @@
           expect(subject.expired?).to eql(false)
         end
       end
+
+      context 'with :from attribute' do
+        subject { Ably::Models::TokenDetails.new(expires: expire_time) }
```

Comment:
That's a little picky, but I'd suggest naming a subject as it improves readability 
```
subject(:token_detail) { .. }
```

Reply:
I can change it but it does not matter for `123` occurrences of other "subject"s in +39 files.

```bash
find . -name "*_spec.rb" -exec grep -H "subject {" {} \; | wc -l   
     123
```